app/views/handlers/block.py

```diff
@@ -120,7 +120,7 @@ def _update_section_completeness(
     def _set_started_at_metadata(self):
         response_metadata = self._questionnaire_store.response_metadata
         if not response_metadata.get("started_at"):
-            started_at = datetime.utcnow().isoformat()
+            started_at = datetime.now(timezone.utc).replace(tzinfo=None).isoformat()
```

Comment:
Replace can be removed.

Reply:
I think it needs to stay as it removes "UTC offset to corresponding time zone" after usage of `.isofromat()` method which returns:
- date
- time
- UTC offset
when we convert `datetime.utcnow()` to iso utc offset is trimmed.

Reply:
I don't see a reason to not keep UTC offset, it is a valid ISO. Worth confirming with Mark as we may need to update the docs around started at in the ons docs.

Reply:
We specify in the docs that "all datetimes are expressed using ISO_8601 and are assumed to be normalised to UTC _unless_ a timezone identifier is given". As Mebin points out the inclusion of the offset is a valid ISO date. This implies that downstream _should_ expect to handle offsets in the value if they are included.

Having said that, convention has dictated that we haven't previously included the offset (as we've never needed to). While the doc doesn't say if its UTC (GMT +0) then we _won't_ include the offset, we would probably need to communicate any change to include the offset to both SDX and processing.